recipes/mingw/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+import os
+from conans import ConanFile, tools
+
+
+class MinGWTestConan(ConanFile):
+    generators = "gcc"
+    settings = "os", "arch", "compiler", "build_type"
+
+    def build(self):
+        source_file = os.path.join(self.source_folder, "main.cpp")
+        self.run("gcc {} @conanbuildinfo.gcc -lstdc++ -o main".format(source_file), run_environment=True)
```

Comment:
I haven't checked this recipe locally, but does this package contain `gcc`?
Most often, mingw compilers have a prefix: e.g. `x86_64-w64-mingw32-gcc` or something similar.